Compression/fcl/prolog.fcl

```diff
@@ -0,0 +1,204 @@
+BEGIN_PROLOG
+
+CompressionTemplates : {
+  compressDetStepMCs_common : {
+    module_type : CompressDetStepMCs
+    strawGasStepTag : "StrawGasStepMaker"
+    caloShowerStepTag : "CaloShowerStepMaker"
+    crvStepTag : "CrvSteps"
+    simParticleTag : "g4run"
+    stepPointMCTags : [ "g4run:virtualdetector", "g4run:protonabsorber" ]#, "g4run:stoppingtarget" ]
+    mcTrajectoryTag : "g4run"
+
+    debugLevel : 0
+  }
+
+  compressDetStepMCs_none : {
+    strawGasStepCompressionLevel : "noCompression" # keep all StrawGasSteps
+    caloShowerStepCompressionLevel : "noCompression"
+    crvStepCompressionLevel : "noCompression"
```

Comment:
Could you configure noCompression automatically, ie if the input tag is set to ""?  That would simplify the scripts.

Reply:
I think there's a difference between noCompression and not having the input collection. For the noCompression option, the idea is to just copy the input collection to an output collection so that it has the correct module label. This way, when debugging, we don't need to change downstream fhicl. 

Your comment does raise the question about what this module will do if we don't create one of the step collections. At the moment, it will throw an exception but maybe we want the flexibility to run with just only some of the detector steps?